package.json

```diff
@@ -68,7 +68,8 @@
     "backbone-query-parameters": "^0.4.0",
     "jquery": "1.11.0",
     "jquery-scrollstop": "^1.2.0",
-    "underscore": "1.4.4"
+    "underscore": "1.4.4",
+    "unveilable": "^0.2.0"
```

Comment:
I've created an npm consumable version of the jQuery `unveil` plugin, which allows us to continue to run our tests in a pure Node environment.